atom/browser/ui/win/atom_desktop_native_widget_aura.cc

```diff
@@ -19,4 +23,22 @@ void AtomDesktopNativeWidgetAura::Activate() {
     views::DesktopNativeWidgetAura::Activate();
 }
 
+void AtomDesktopNativeWidgetAura::OnWindowActivated(
+    wm::ActivationChangeObserver::ActivationReason reason,
+    aura::Window* gained_active,
+    aura::Window* lost_active) {
+  views::DesktopNativeWidgetAura::OnWindowActivated(
+    reason, gained_active, lost_active);
+  if (lost_active != nullptr) {
+    auto* tooltip_controller = static_cast<views::corewm::TooltipController*>(
+      wm::GetTooltipClient(lost_active->GetRootWindow()));
+
+    // This will cause the tooltip to be hidden when a window is deactivated,
+    // as it should be.
+    // TODO(brenca): Remove this fix when the chromium issue is fixed.
+    // crbug.com/724538
+    tooltip_controller->OnCancelMode(nullptr);
```

Comment:
Is there any way for `wm::GetTooltipClient()` to return `nullptr` such that this would be a null pointer dereference?

Reply:
That could return `NULL`, 'tho I don't really see when. Let's add an if just in case.